torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2892,7 +2884,7 @@ def call_function(
                 ),
                 kwargs={},
             ),
-            example_value=example_value,
+            example_value=None,
```

Comment:
By passing in None for example value ` wrap_fx_proxy` will implicitly do the above